source/_integrations/trafikverket_ferry.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: Trafikverket Ferry
+description: Instructions how to integrate Trafikverket Ferry within Home Assistant.
+ha_category:
+  - Sensor
+  - Transport
+ha_release: 2022.5
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@gjohansson-ST'
+ha_domain: trafikverket_ferry
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Retrieve ferry departure information from [Trafikverket](https://www.trafikverket.se/).
+
+## Prerequisites
+
+Please click [here](https://api.trafikinfo.trafikverket.se/) and register to obtain the API key.
+
+{% include integrations/config_flow.md %}
+
+## Use cases
+
+- Retrieve the next departure and information connected to it
+- Destination harbour is optional and by leaving empty it will show next destination available from selected departure harbour.
+- By choosing a specific time it will only show departures after this.
+- By deselecting from the pre-filled days of week you can make selection such as only Monday to Friday if needed.
```

Comment:
```suggestion
## Use cases

- Retrieve the next departure and information connected to it.
- Destination harbor is optional and by leaving it empty it will show the next destination available from the selected departure harbor.
- By choosing a specific time it will only show departures after this.
- By deselecting from the pre-filled days of the week you can make selections such as only Monday to Friday if needed.
```